core/services/keeper/upkeep_executer.go

```diff
@@ -207,24 +205,28 @@ func (executer *UpkeepExecuter) execute(upkeep UpkeepRegistration, headNumber in
 		runErrors = pipeline.RunErrors{null.StringFrom(errors.Wrap(err, "UpkeepExecuter: failed to construct upkeep txdata").Error())}
 	}
 
-	var etx bulletprooftxmanager.EthTx
-	ctxQuery, cancel := postgres.DefaultQueryCtx()
+	// postgres.DefaultQueryCtx() returns the same context with the same deadline.
+	// Just to avoid mixing store layer and service one the default context is used here.
+	txCtx, cancel := context.WithTimeout(context.Background(), time.Second*10)
```

Comment:
hmm but `postgres.DefaultQueryCtx()` is a commonly used default - we were moving in the direction of using it more. Perhaps we could move / copy it to another package like postgresUtils, etc?

Reply:
I guess it is not really the right way to use "postgres" package in the business layer, transport layer etc.  We could move this func to another package that is not related to ORM.

Reply:
alternatively we could move part of this method to keeper/orm.go

Reply:
Yea, I just don't want to have a big PR so it is better to do improvements by small portions

Reply:
Let's do the following: use postgres context for now, but one of next PRs will include this refactoring

Reply:
ok - sounds good